gen.go

```diff
@@ -97,6 +99,21 @@ var goversions = map[string]*GoVersion{
 }
 `))
 
+var client = &http.Client{}
+
+var authRequest func(*http.Request)
+
+func init() {
+	token := os.Getenv("GITHUB_TOKEN")
+	if token != "" {
+		authRequest = func(r *http.Request) {
+			r.Header.Set("Authorization", "token "+token)
```

Comment:
Should be `Bearer` instead of `token`. https://docs.github.com/en/actions/security-guides/automatic-token-authentication#example-2-calling-the-rest-api